### PR TITLE
[V2V] Remove SetMigrated state. Cleanup state machine.

### PR DESCRIPTION
**Pull request by fdupont-redhat**
The state machine is being ported to InfraConversionJob.
This PR is the counterpart of ManageIQ/manageiq#19288.

RHBZ: https://bugzilla.redhat.com/show_bug.cgi?id=1751350
Depends on ManageIQ/manageiq#19288

### CODE REVIEW

**Comment by fdupont-redhat** [comment]
@miq-bot add-label transformation, refactoring, ivanchuk/yes
@miq-bot add-reviewer @tinaafitz 
@miq-bot add-reviewer @gmcculloug

File: content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml [hunk 1]
```diff
@@ -10,25 +10,9 @@ object:
...
       value: "/Transformation/Common/WaitForHandover"
-      on_entry: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
-      on_exit: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
-      on_error: /System/CommonMethods/MiqAe.WeightedUpdateStatus(weight => 1, description
-        => "Waiting for Handover", task_message => "Pre-migration")
+      on_entry: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
+      on_exit: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
+      on_error: "/System/CommonMethods/MiqAe.WeightedUpdateStatus"
```
Comment (gmcculloug) [inline content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml]: @fdupont-redhat Why are you retaining these calls?  What does the method do when passed without parameters?

Comment (fdupont-redhat) [reply content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml]: The method should do nothing but allow cancellation, which would be useless at this step. I removed the calls.

Comment (gmcculloug) [reply content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml]: @fdupont-redhat Which calls did you remove?  These are still "mostly" here.  🙂

Comment (fdupont-redhat) [reply content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml]: Hum. Weird. When I look at 'Files changed' tab, I can see they are gone.
Grep'ing `WeightedUpdateStatus` in the repo only return the definition of the method under /System/CommonMethods/MiqAe.

Comment (gmcculloug) [reply content/automate/ManageIQ/Transformation/StateMachines/VMTransformation.class/transformation.yaml]: I'm seeing all deletes now.  Not sure if I had a stale page open before.  👍

**Comment by simaishi** [comment]
Ivanchuk backport details:

```
$ git log -1
commit 17ef960a6fc026336b1521c4bbaeb02614b9cec4
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Sep 17 11:21:58 2019 -0400

    Merge pull request #579 from fdupont-redhat/v2v_state_machine_remove_cancellation
    
    [V2V] Remove SetMigrated state. Cleanup state machine.
    
    (cherry picked from commit b942a2d4e6bb1a9ca9034ac4342821c13e0bb678)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1768528
```

**Comment by simaishi** [comment]
The backport reverted

```
commit 66ec97e8bdb8adff7b9e8340df2564df14491997
Author: Satoe Imaishi <simaishi@redhat.com>
Date:   Tue Dec 3 11:10:55 2019 -0500

    Revert "Merge pull request #579 from fdupont-redhat/v2v_state_machine_remove_cancellation"

    This reverts commit 17ef960a6fc026336b1521c4bbaeb02614b9cec4.

    https://bugzilla.redhat.com/show_bug.cgi?id=1768528
```

**Comment by simaishi** [comment]
Ivanchuk backport details:

```
$ git log -1
commit 9e666485f5b3aa50b085be9e67aa7fcdedc438b7
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Tue Sep 17 11:21:58 2019 -0400

    Merge pull request #579 from fdupont-redhat/v2v_state_machine_remove_cancellation

    [V2V] Remove SetMigrated state. Cleanup state machine.

    (cherry picked from commit b942a2d4e6bb1a9ca9034ac4342821c13e0bb678)

    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1768528
```

**Pull request merged**